lib/app_profiler/viewer/speedscope_remote_viewer.rb

```diff
@@ -17,9 +17,15 @@ def initialize(profile)
         @profile = profile
       end
 
-      def view
+      def view(response: nil, autoredirect: nil, async: false)
```

Comment:
Because we're passing in the parameters from the view action (from the middleware), we can expect these to be provided. If we don't mention them, Ruby will raise an ArgumentError and our public API will be broken if we don't provide defaults.